test/jdk/java/awt/Component/ComponentEventTest.java

```diff
@@ -41,12 +41,15 @@
 import java.awt.event.ComponentListener;
 import java.awt.event.InputEvent;
 import java.lang.reflect.InvocationTargetException;
+import jdk.test.lib.Platform;
```

Comment:
```suggestion
import java.lang.reflect.InvocationTargetException;

import jdk.test.lib.Platform;
```

You should put a blank line to separate standard library imports from custom (internal) classes.